GoogleForCreators/web-stories-wp
# Templates: use custom heading levels

## Context



## Summary
Updates templates to use custom heading levels were necessary


## Relevant Technical Choices
- Following the automated headings rule, texts with less than 3 characters are not assigned headings
- There can be multiple h1 tags per story
- The first page always has a h1 now



## To-do



## User-facing changes
N/A


## Testing Instructions






This PR can be tested by following these steps:

1. Go to Explore Templates
2. Select "Use Template" for different templates
3. Click through the text elements in the story that opens to see if the heading levels assigned seem ok


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #11928
